tck/src/main/resources/tests/errorHandling/invalidDataTypeValue/output.json

```diff
@@ -1,12 +1,14 @@
 {
-  "data": null,
+  "data": {
+    "updateItemPowerLevel": null
```

Comment:
Yes, because the validation is not handles with GraphQL-java anymore, but later in the chain (by us) as we can do formatting. We can maybe relax the check if sqpr impl handle this differently.